jdk_17_maven/cs/rest/familie-ba-sak/pom.xml

```diff
@@ -392,6 +392,10 @@
             <id>github</id>
             <url>https://maven.pkg.github.com/navikt/familie-felles</url>
         </repository>
+        <repository>
+            <id>familie</id>
+            <url>https://github-package-registry-mirror.gc.nav.no/cached/maven-release</url>
+        </repository>
```

Comment:
why modifying this?

Reply:
Hi Andrea,

Thanks for your comment. The mvn command failed to fetch familie-ba-sak in my machine (MacOS). I saw the repository in the link https://github.com/navikt/github-package-registry-mirror. It worked in my case after adding this repository.

Reply:
@onurd86 was this before or after fixing your `settings.xml` file?

Reply:
@arcuri82 This was after fixing settings.xml file.